src/Makevars

```diff
@@ -1,4 +1,4 @@
 CXX_STD = CXX17
-PKG_CXXFLAGS = -std=c++17  -DTMB_MODEL  -DTMB_EIGEN_DISABLE_WARNINGS -w
+PKG_CXXFLAGS = -DTMB_MODEL  -DTMB_EIGEN_DISABLE_WARNINGS
 CXX17STD = -std=c++17 -w
-
+USE_CXX17 = "yes"
```

Comment:
does this line need to be added to Makevars.win?

Reply:
The R CMD check for windows, linux, and mac currently states
```
* checking C++ specification ... OK
  Not all R platforms support C++17
```
with the current configuration so I believe the files are fine as is.